packages/sitecore-jss-react/src/components/FEaaSComponent.test.tsx

```diff
@@ -166,5 +166,27 @@ describe('<FEaaSComponent />', () => {
         `data="${props.params?.ComponentDataOverride!.replace(/"/g, '&quot;').replace(/\s/g, '')}"`
       );
     });
+
+    it('should get data using SSR', () => {
```

Comment:
Let's call it a bit similarly to what we have above: [test](https://github.com/Sitecore/jss/pull/1590/files#diff-3380d84bec90ed0b3b1109f56aa1a0b4c2909f110d9a8bae00500554d0459497R85)
```suggestion
    it('should send prefetched data ', () => {
```